lib/generators/draper/templates/application_decorator.rb

```diff
@@ -0,0 +1,11 @@
+class ApplicationDecorator < Draper::Decorator
+  delegate_all
+
+  # Define methods for all decorated objects.
+  # Helpers are accessed through `helpers` (aka `h`). For example:
+  #
+  #   def percent_amount
+  #     h.number_to_percentage object.amount, precision: 2
+  #   end
+
```

Comment:
@codebycliff, I was basing this template off of the [decorator template](https://github.com/drapergem/draper/blob/master/lib/generators/rails/templates/decorator.rb).  Do you want the two to match?